data/templates/styles.html

```diff
@@ -161,6 +161,16 @@
 $if(csl-entry-spacing)$
   margin-bottom: $csl-entry-spacing$;
 $endif$
+$if(abstract)$
+p.abstract{
+  text-align: center;
+  font-weight: bold;
```

Comment:
If we do include this, I'd prefer a more vanilla formatting.  Normally abstracts in journals are not boldface.